releases/release-4.0.13.md

```diff
@@ -0,0 +1,128 @@
+    - Fix wrong flen infer for bit constant [#24266](https://github.com/pingcap/tidb/pull/24266)
+    - Fix an issue that `NO_ZERO_IN_DATE` SQL mode does not work in default value [#24185](https://github.com/pingcap/tidb/pull/24185)
+    - Fix type merge about bit type. [#24026](https://github.com/pingcap/tidb/pull/24026)
+    - Planner: fix wrong TableDual plans caused by comparing Binary and Bytes incorrectly [#23917](https://github.com/pingcap/tidb/pull/23917)
+    - Fix insert ignore on duplicate may delete wrong record [#23825](https://github.com/pingcap/tidb/pull/23825)
+    - Plugin: fix audit plugin will cause tidb panic [#23819](https://github.com/pingcap/tidb/pull/23819)
+    - Fix collation for hash join building [#23812](https://github.com/pingcap/tidb/pull/23812)
+    - Fix batch point lock maybe panic when condition value overflow [#23778](https://github.com/pingcap/tidb/pull/23778)
+    - Fix text type decode for old row format [#23772](https://github.com/pingcap/tidb/pull/23772)
+    - Fix bug when compare int_column with constant value [#23705](https://github.com/pingcap/tidb/pull/23705)
+    - Fix approx_percent panic on bit column [#23702](https://github.com/pingcap/tidb/pull/23702)
+    - Fix a bug that causes TiDB to report `TiKV server timeout` when executing TiFlash batch request. [#23700](https://github.com/pingcap/tidb/pull/23700)
+    - Fix index join on prefix column index [#23691](https://github.com/pingcap/tidb/pull/23691)
+    - Fix collation for binary literal [#23598](https://github.com/pingcap/tidb/pull/23598)
+    - Fix update panic on join having statement. [#23575](https://github.com/pingcap/tidb/pull/23575)
+    - Planner: fix inappropriate null flag of null constants [#23474](https://github.com/pingcap/tidb/pull/23474)
+    - Fix unexpected constant fold when year compare string. [#23335](https://github.com/pingcap/tidb/pull/23335)
+    - Group_concat aggr panic when session.group_concat_max_len is small. [#23257](https://github.com/pingcap/tidb/pull/23257)
+    - Fix incorrect duration between compare [#23233](https://github.com/pingcap/tidb/pull/23233)
+    - Fix the delete stmt privilege check. [#23215](https://github.com/pingcap/tidb/pull/23215)
+    - Types: fix the bug about the wrong query result for decimal type [#23196](https://github.com/pingcap/tidb/pull/23196)
+    - Fix wrong error info [#23152](https://github.com/pingcap/tidb/pull/23152)
+    - Fix wrong index merge selection [#22924](https://github.com/pingcap/tidb/pull/22924)
+    - Fix enum and set type expression in where clause [#22814](https://github.com/pingcap/tidb/pull/22814)
+    - Fix a bug which causes panic when using the clustered index and the new collation [#21408](https://github.com/pingcap/tidb/pull/21408)
+    - Fix the panic when analyze with collation enabled [#21299](https://github.com/pingcap/tidb/pull/21299)
+
++ TiFlash
+
```

Comment:
- Fix the issue that number of `delta-merge-tasks` is not reported to Prometheus
    - Fix the TiFlash panic issue that occurs during `Segment Split`
    - Fix the issue that panel `Region write Duration (write blocks)` in Grafana is shown in wrong place
    - Fix the potential issue that `DeleteRange` in storage engine fail to remove data
    - Fix the issue of incorrect results when casting the time type to the integer type
    - Fix a bug that the behavior of the bitwise operator is different from that of TiDB
    - Fix the issue of incorrect results when casting the string type to the integer type
    - Fix the potential issue that continuous and fast writing make TiFlash OOM
    - Fix the TiFlash panic issue that occurs when writing data to dropped tables
    - Fix the potential issue that the exception of null pointer might be raised during the table GC
    - Fix the issue that TiFlash might panic during BR restore
    - Fix a bug that weights of some characters are wrong when using general CI collation
    - Fix the potential issue that data will be lost in tombstoned table
    - Fix the issue of incorrect results when comparing string which contains zero bytes
    - Fix the issue that logical function returns wrong result if input column contains null constants
    - Fix the issue that logical function only accept numeric type
    - Fix the issue of incorrect results when timestamp value is `1970-01-01` and the timezone offset is negative
    - Fix the issue that hash value of Decimal256 is not stable